components/centraldashboard/public/components/registration-page.js

```diff
@@ -103,7 +103,7 @@ export class RegistrationPage extends utilitiesMixin(PolymerElement) {
         if (!this.validateNamespace()) return;
         API.body = {namespace: this.namespaceName};
         this.waitForRedirect = true;
-        await API.generateRequest().completes;
+        await API.generateRequest().completes.catch((e) => {});
```

Comment:
* Don't name unused variables. You can `() => {}`, or name the param `_`.
* Add a comment that makes it clear we do nothing with the rejection because the error is handled elsewhere.